src/extensions/WorkspaceView/sortingExamples/spykingcircus_example.py

```diff
@@ -0,0 +1,45 @@
+# Prerequisites:
```

Comment:
Example spyking circus script that appears in the spike sorting instructions panel.